const.go

```diff
@@ -113,13 +110,13 @@ var ALL_STATUSES = []string{
 
 // incomplete until all statuses are implemented
 var VALID_STATUS_TRANSITIONS = [][]string{
-	[]string{STATUS_PENDING, STATUS_ACTIVE},
-	[]string{STATUS_ACTIVE, STATUS_PAUSED},
-	[]string{STATUS_PAUSED, STATUS_ACTIVE},
-	[]string{STATUS_PENDING, STATUS_RESOLVED},
-	[]string{STATUS_PAUSED, STATUS_RESOLVED},
-	[]string{STATUS_ACTIVE, STATUS_RESOLVED},
-	[]string{STATUS_PENDING, STATUS_TEMPLATE},
+	{STATUS_PENDING, STATUS_ACTIVE},
```

Comment:
Hm, didn't know this was allowable.

Reply:
Me either, but `golint` and my vim plugin were shouting at me about it.